drivers/gpu/drm/drm_connector.c

```diff
@@ -362,7 +362,7 @@ static int __drm_connector_init(struct drm_device *dev,
 	drm_object_attach_property(&connector->base,
 				   config->non_desktop_property,
 				   (connector_type != DRM_MODE_CONNECTOR_VIRTUAL &&
-				   connector_type != DRM_MODE_CONNECTOR_WRITEBACK) ? 0 : 1;
+				   connector_type != DRM_MODE_CONNECTOR_WRITEBACK) ? 0 : 1);
```

Comment:
This should be squashed in previous commit.

Reply:
Done.